### ISSUE TITLE
Inconsistency with `use-fstring-for-concatenation`

### ISSUE THREAD

**Issue by diceroll123** [sourcery-ai/sourcery#222]
### Issue description or question

I can't exactly put my finger on what's going wrong here, y'all must be doing some weird stuff behind the scenes.

```py
def _test_sourcery_thing():
    # does NOT suggest use-fstring-for-concatenation
    # let's call this the original function
    favicon_url = "https://github.com/"
    return "https://s2.googleusercontent.com/s2/favicons?domain_url=" + favicon_url

def _test_sourcery_thing1():
    # works as expected
    # difference from original is the name of the variable
    url = "https://github.com/"
    return "https://s2.googleusercontent.com/s2/favicons?domain_url=" + url

def _test_sourcery_thing2():
    # works as expected 
    # difference from original is both strings are empty
    favicon_url = ""
    return "" + favicon_url
```

Hope this helps!

### Sourcery Version

v0.11.2

### Code editor or IDE name and version

Pycharm v?

### OS name and version

Version: 1.66.0 (user setup)
Commit: e18005f0f1b33c29e81d732535d8c0e47cafb0b5
Date: 2022-03-30T05:50:14.623Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Windows_NT x64 10.0.22593

**Comment by ruancomelli** [comment]
Hi, @diceroll123, thank you for this submitting this bug report! Keep them coming :grin: 

I was able to reproduce this and will look into it. And yes, replacing `f"{string}"` with simply `string` is a great idea, I've added it to our roadmap.

**Comment by ruancomelli** [comment]
Hi again! I just added a fix for the issue you reported - it'll be available in the next Sourcery release.

**Comment by Hellebore** [comment]
Fixed in 0.11.3
